src/Client.php

```diff
@@ -39,7 +39,6 @@ class Client
      */
     public function __construct($config)
     {
-        $config = Config::ensure($config);
```

Comment:
Why not propose both options ? Just passing-in an array it probably the default case. In some other cases the user might want to build the Config object in another place before passing it to the client.